src/table/utils/styling-utils.js

```diff
@@ -54,7 +57,7 @@ export const getBaseStyling = (styleObj, objetName, theme) => {
   const baseStyle = {
     backgroundColor: theme.table.backgroundColor,
     fontFamily,
-    color: styleObj?.fontColor ? getColor(STYLING_DEFAULTS.FONT_COLOR, theme, styleObj.fontColor) : color,
+    color: isSet(styleObj?.fontColor) ? getColor(STYLING_DEFAULTS.FONT_COLOR, theme, styleObj.fontColor) : color,
```

Comment:
Is there a test on these two cases?

Reply:
Good question. Do not think there is any. Let me add some.

Reply:
Done